src/app/page/Missions/Missions.jsx

```diff
@@ -19,14 +20,27 @@ import UserPhoneUnverifiedPopup from "../../component/UserPhoneUnverifiedPopup";
  */
 const MissionsPage = ({ auth, history, missions }) => {
   const [popupOpen, setPopupOpen] = useState(false);
+  const [snackbarOpen, setSnackbarOpen] = useState(false);
```

Comment:
We can probably combine the state properties for the snackbar into an object:
`const initialSnackbarState = {open: false, type: 'error', message: ""}
const [snackbarState, setState] = useState(initialSnackbarState);
...
setState({open:true, type: error, message: "Message"});
`

Reply:
Yeah that makes sense. Will correct it.